doc/source/f2py/code/pyproj_skbuild.toml

```diff
@@ -1,5 +1,3 @@
-[project]
-requires-python = ">=3.7"
```

Comment:
Setuptools (and therefore the current scikit-build) do not read the pyproject file (or any toml). SO this does nothing (sadly)

Reply:
It seems that it is [still relevant for](http://www.scikit-build.org/en/stable/usage.html#support-for-isolated-build) `pip`?

Reply:
Pip reads the `build-system` section. But your build tool reads the project section, and setuptools does not. The Requires-Python metadata slot gets filled by the build system, not by pip.